yt/utilities/exceptions.py

```diff
@@ -91,8 +91,24 @@ def __init__(self, field, ds):
         self.field = field
         self.ds = ds
 
+    def _enumerate_alternatives(self):
```

Comment:
Note for reviewers: this should be part of #2967 if anything.

Reply:
Actually should be part of https://github.com/yt-project/yt/pull/3184. I'm removing it.